src/stylesheets/common/_colors.scss

```diff
@@ -57,46 +57,75 @@ $okColor:#63a240;
 
 
 //Classes for styleguide
```

Comment:
Originally wrote them specifically for styleguide to display the full palette. Happy to rename if we think we will use them elsewhere?
Moving them to styleguidepage.scss 👍